### ISSUE TITLE
Compare and integrate latest changes to MS Transfer module

### ISSUE THREAD

**Issue by amaltaro** [dmwm/WMCore#9138]
We need to check what were the changes done to the Unified transferor algorithm (over the last 18 months or so) and integrate those into the MicroService Transferor module.

Maybe @vlimant can give us some pointers as well.

**Comment by vlimant** [comment]
https://github.com/CMSCompOps/WmAgentScripts/commits/master/Unified/transferor.py there isn't much important business it seems

**Comment by amaltaro** [comment]
I'm inclined to close this issue and follow up whatever needs to be done with a more specific issue. We have learned a few more things over the last couple of months that were either discussed and or integrated into MSTransferor.

**Comment by amaltaro** [comment]
Now that I'm going through the code once again, I noticed that this piece is still missing in the MS:
```
                p = max_priority.get(d, 80000)
                q = 'normal'
                if p > 100000:
                    q = 'reserved'
                elif p < 70000:
                    q = 'low'
```

in plain english. Workflows with `RequestPriority>100k` should be subscribed under the `reserved` phedex priority; and so on.

@ericvaandering is this also a reality in Rucio? Can we define different priorities for the rucio rules? If so, then I think this feature needs to be implemented. Otherwise we can likely live with `normal` for a few months.

**Comment by ericvaandering** [comment]
There are different Activity codes and I think we can prioritize them in Rucio and/or FTS. So something like this may be useful.

Be sure to keep the hardcoded values and single character variable names. :-)

**Comment by vkuznet** [comment]
@amaltaro , can we close this issue?

**Comment by amaltaro** [comment]
@vkuznet yes, we might want to have different rule priority. But we can have a specific issue on that whenever people see a real need for that. Closing this out! Thanks
